cryocare/protocols/protocol_denoise.py

```diff
@@ -49,40 +49,37 @@ def _defineParams(self, form):
         """
         # You need a params to belong to a section:
         form.addSection(label=Message.LABEL_INPUT)
-        form.addParam('message', params.StringParam,
-                      default='Hello world!',
-                      label='Message', important=True,
-                      help='What will be printed in the console.')
+        form.addParam('evenTomo', params.PointerParam,
+                      pointerClass='Tomogram',
+                      label='Tomogram 1 (even)',
+                      important=True,
+                      help='Tomogram reconstructed from the even frames of the tilt'
+                           'series movies.')
 
-        form.addParam('times', params.IntParam,
-                      default=10,
-                      label='Times', important=True,
-                      help='Times the message will be printed.')
+        form.addParam('oddTomo', params.PointerParam,
+                      pointerClass='Tomogram',
+                      label='Tomogram 1 (odd)',
```

Comment:
Tomogram 2?

Reply:
For denoising.....do we actually need the odd and even ones? or just a noisy tomogram.

Reply:
I can see we do need them.

Reply:
If it's even and odd halves, it must be tomo 1 (same tomogram) otherwise it will be a disaster.

Reply:
Maybe just removing the number will do. Actually there is a chance that we can input a set of tomograms and avoid in the denoise the odd/even stuff, leaving it to just the training. Depends on the results